src/pages/storage/forms/StoragePoolFormMain.tsx

```diff
@@ -46,6 +48,12 @@ const StoragePoolFormMain: FC<Props> = ({ formik }) => {
   const isPowerFlexDriver = formik.values.driver === powerFlex;
   const isPureDriver = formik.values.driver === pureStorage;
   const storageDriverOptions = getStorageDriverOptions(settings);
+  const hasClusterWideSource = isCephDriver || isCephFSDriver;
+  const hasSource = !isPureDriver && !isPowerFlexDriver;
+
+  const helpText = formik.values.isCreating
```

Comment:
```suggestion
  const sourceHelpText = formik.values.isCreating
```
This component handles other fields, better to be specific.